blockly-ws-search/README.md

```diff
@@ -0,0 +1,32 @@
+[Home](../README.md)
```

Comment:
Remove this link, since this is a published package, we don't want this around in the npm readme